src/contracts/pods/EigenPod.sol

```diff
@@ -55,6 +58,16 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     /// @notice The amount of eth, in wei, that is restaked per ETH validator into EigenLayer
     uint256 public immutable REQUIRED_BALANCE_WEI;
```

Comment:
Adding a note here for sake of tracking.
We discussed offline the possibility of:

- rename this variable to something like `MAX_RESTAKED_AMOUNT_PER_VALIDATOR` and clarify in the documentation that this is how it is being used
- get rid of the check against this when proving withdrawal credentials, and instead require that the proof come from a recent block number -- this simplifies the design and addresses some concerns about people being able to prove super out-of-date validator balances, and AFAICT people could still go through the normal flow, since we now treat the restaked amount more dynamically -- we had been allowing users to prove withdrawal credentials from any time (even far in the past) to ensure that if they had withdrawn (also possibly far in the past) that they could provide a proof which satisfied this check, but if we remove the check then we can use the same slot-staleness check as we have when proving balance updates (so this change would hopefully result in more consistent treatment of all validators)